### PR TITLE
Update importxml.py

### PR DESCRIPTION
**Pull request by PushKK**
1. Change 2 whitespaces to '\t' (for Weblate),
2. Change ": " to COLON,
3. Change strings 'Xml' and 'xml' to 'XML'.

Using '\t' help to create aligned text. You can compare before and after:
ImportStatistics
Снимок экрана_2021-06-05_16-06-58

### CODE REVIEW

File: gramps/plugins/importer/importxml.py [hunk 1]
```diff
@@ -279,21 +278,21 @@ def info_text(self):
...
-            FAMILY_KEY      : _('  Families: %d\n'),
-            SOURCE_KEY      : _('  Sources: %d\n'),
-            EVENT_KEY       : _('  Events: %d\n'),
-            MEDIA_KEY       : _('  Media Objects: %d\n'),
-            PLACE_KEY       : _('  Places: %d\n'),
-            REPOSITORY_KEY  : _('  Repositories: %d\n'),
-            NOTE_KEY        : _('  Notes: %d\n'),
-            TAG_KEY         : _('  Tags: %d\n'),
-            CITATION_KEY    : _('  Citations: %d\n'),
+            PERSON_KEY      : _('\tPeople{colon}\t%d\n').format(colon=COLON),
```
Comment (Nick-Hall) [inline gramps/plugins/importer/importxml.py:281]: Using tabs in the middle of a string like this doesn't really work.

File: gramps/plugins/importer/importxml.py [hunk 2]
```diff
@@ -279,21 +278,21 @@ def info_text(self):
...
+            REPOSITORY_KEY  : _('\tRepositories{colon}\t%d\n').format(colon=COLON),
+            NOTE_KEY        : _('\tNotes{colon}\t%d\n').format(colon=COLON),
+            TAG_KEY         : _('\tTags{colon}\t%d\n').format(colon=COLON),
+            CITATION_KEY    : _('\tCitations{colon}\t%d\n').format(colon=COLON),
             }
         txt = _("Number of new objects imported:\n")
         for key in self.keyorder:
             if any(self.data_unknownobject):
-                strng = key2string[key][0:-1] + ' (%d)\n'
+                strng = key2string[key][0:-1] + '\n(%d)\n'
```
Comment (Nick-Hall) [inline gramps/plugins/importer/importxml.py:295]: Why do you want the unknown object count on a new line?

There is also a problem for RTL languages with using string concatenation here.  I'll fix it in a separate PR.

**Review by Nick-Hall — CHANGES_REQUESTED** [review]
The XML changes look OK.

File: gramps/plugins/importer/importxml.py (hunk 1 above)
Comment (PushKK) [reply gramps/plugins/importer/importxml.py:281]: In the picture above you can see work this tabs.
Using \t I can align digit after text.

**Comment by Nick-Hall** [comment]
I see the following with this patch:
ImportScreenshot

File: gramps/plugins/importer/importxml.py (hunk 2 above)
Comment (PushKK) [reply gramps/plugins/importer/importxml.py:295]: Oh, it's my error. Sorry...

**Comment by PushKK** [comment]
> I see the following with this patch:
Oh, you not use magic in .po file (for en_US not available).
From ru.po:
```
#~ msgid "Media Objects{colon}\t%d"
#~ msgstr "Документов{colon}\t%d"

#~ msgid "Places{colon}\t%d"
#~ msgstr "Мест{colon}\t\t\t%d"
```
I need add '\t' in tranlated text to align digit after words of different lengths:
Документов\t
Мест\t\t\t

File: gramps/plugins/importer/importxml.py (hunk 1 above)
Comment (PushKK) [reply gramps/plugins/importer/importxml.py:281]: Translator add needed '\t' in .po file for align word in his language.

**Comment by Nick-Hall** [comment]
This has been replaced by PR #1259 which adds the ability to display simple tables in information dialogs.

I'll close this now.

**Pull request closed without merging**